LedgerSMB/Scripts/contact.pm

```diff
@@ -340,7 +341,7 @@ sub _main_screen {
                    person => $person,
                  employee => $person,
                      user => $user,
-                    roles => eval { $user->list_roles },
```

Comment:
Right. I did it the way I did, because that reduces the scope of the exception handling suppression (that's basically the effect of ignoring $@)